lib/former_employee.rb

```diff
@@ -0,0 +1,27 @@
+class FormerEmployee
```

Comment:
Nice to see a null object here.  One problem is that not all authors who don't appear in the `data.people` are ex employees - we've had some guest bloggers before and maybe even clients?  So perhaps `NonEmployee` would be more correct (albeit not as snappy, better names are possible I'm sure).